### DISCUSSION TITLE
Is there a way to remove `Register` button from the `SignInScreen`?

### DISCUSSION THREAD

**Discussion by muddi900** [firebase/FirebaseUI-Flutter#348] [Q&A]
I have a module where only login is required.

Is it possible?

**Comment by StevenSnary** [comment]
I believe this is what you are looking for - set the showAuthActionSwitch parameter to false.

`.SignInScreen(showAuthActionSwitch: false)`

I've done this in the web version of my app as I want the user to be able to register in the iOS or Android version, but the web version is only to be used to users that have already registered.

This is what my 'No Register' SignInScreen looks like...

<img width="944" alt="Screenshot 2024-06-10 at 3 08 06 PM" src="https://github.com/firebase/FirebaseUI-Flutter/assets/10373429/e5b14487-ff6b-4b10-9089-102d8403650c">
